keyboards/lets_split/keymaps/tylerwince/keymap.c

```diff
@@ -0,0 +1,239 @@
+ * |Things|1PASS | alt  | CMD  |Lower |Space |   |Space |Raise | Next | Vol- | Vol+ | Play |
+ * `------------------------------------------   ------------------------------------------'
+ */
+[_RAISE] = LAYOUT_ortho_4x12(
+  KC_GRV,       KC_1,          KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,     KC_0,    KC_DEL,
+  KC_DEL,       _______,       _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT, _______, KC_BSLS,
+  _______,      _______,       _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,
+  RALT(KC_SPC), RGUI(KC_BSLS), KC_LALT, KC_LCMD, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_MNXT, KC_VOLD, KC_VOLU,  KC_MPLY
+),
+
+/* Adjust (Lower + Raise)
+ * ,------------------------------------------   ------------------------------------------.
+ * |      | Reset|      |      |      |      |   |      |Qwerty|Colemk|Dvorak|WTLEFT|WTRGHT|
+ * |------+------+------+------+------+-------   -------+------+------+------+------+------|
+ * |      |      |Aud on|Audoff|AGnorm|AGswap|   |WLEFT |WDOWN | WUP  |WRGHT |WBLEFT|WBRGHT|
+ * |------+------+------+------+------+------|   |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |   |      |RGBsUP|RGBsDN|RGBvUP|RGBvDN|WFULL |
+ * |------+------+------+------+------+------+   +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |   |      |      |RGBwav|RGBfla|RGBtog|      |
+ * `------------------------------------------   ------------------------------------------'
+ */
+[_ADJUST] =  LAYOUT_ortho_4x12(
+  _______, RESET,   _______, _______, _______, _______, _______,   QWERTY,    COLEMAK,   DVORAK,    LCA(KC_7), LCA(KC_8),
+  _______, _______, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, LCA(KC_H), LCA(KC_J), LCA(KC_K), LCA(KC_L), LCA(KC_U), LCA(KC_I),
+  _______, _______, _______, _______, _______, _______, _______,   RGB_SAI,   RGB_SAD,   RGB_VAI,   RGB_VAD,   LCA(KC_ENT),
+  _______, _______, _______, _______, _______, _______, _______,   _______,   RGB_M_K,   RGB_M_B,   RGB_TOG,   _______
+)
+
+
+};
+
+void keyboard_post_init_user(void) {
+  #ifdef RGBLIGHT_ENABLE
+  rgblight_enable_noeeprom();
+  layer_state_set_user(layer_state);
+  #endif
+}
+void set_qwerty(void){
+  rgblight_sethsv_noeeprom(255, 0, 160);
+}
+void set_lower(void){
+  rgblight_sethsv_master(255, 255, 255);
+  rgblight_sethsv_slave(255, 0, 160);
+}
+void set_raise(void){
+  rgblight_sethsv_slave(35, 255, 255);
+  rgblight_sethsv_master(255, 0, 160);
+}
+void set_adjust(void){
+  rgblight_sethsv_slave(35, 255, 255);
+  rgblight_sethsv_master(255, 255, 255);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef RGBLIGHT_ENABLE
+          set_qwerty();
+        #endif
+        set_single_persistent_default_layer(1UL<<_QWERTY);
```

Comment:
`set_single_persistent_default_layer()` just takes the layer index/name directly; no bit shifting needed.

```suggestion
        set_single_persistent_default_layer(_QWERTY);
```